internal/engine/upper_test.go

```diff
@@ -1729,37 +1737,52 @@ func TestDockerComposeEventSetsStatus(t *testing.T) {
 	})
 }
 
-func TestDockerComposeRecordsLogs(t *testing.T) {
+func TestDockerComposeRecordsBuildLogs(t *testing.T) {
 	f := newTestFixture(t)
 	m, _ := f.setupDCFixture()
-	expected := "spoonerisms_1  | 2018-12-20T16:11:04.070480042Z yarn install v1.10."
-	f.dcc.SetLogOutput(expected + "\n")
+	expected := "yarn install"
+	f.setBuildLogOutput(m.ManifestName(), expected)
 
-	f.Start([]model.Manifest{m}, true)
-	f.waitForCompletedBuildCount(1)
+	f.loadAndStart()
+	f.waitForCompletedBuildCount(2)
```

Comment:
wait why are there two builds now?

Reply:
The docker compose fixture now has two services ("redis" and "server), before it only had one.

Reply:
oh. just call `f.Start([]model.Manifest{m}, true)`, we've already loaded above. there are two for the cases where they're needed/useful, but should probs just use one unless it's a specific test for >1 manifest.